src/Input/InputEvents.cpp

```diff
@@ -479,3 +491,17 @@ InputEvents::ProcessTimer()
   DoQueuedEvents();
   ProcessMenuTimer();
 }
+
+void  //////// LOCKSCREEN
+InputEvents::eventLockscreen(const TCHAR *mode)
+{
+  LockScreen = !LockScreen;
+  if (LockScreen){
```

Comment:
What does this mean?

Reply:
It's to flip the variable that stores the enabling status of the lockscreen function

Reply:
But ... why? Why does there even need to be a "status"? Why flip?

Reply:
I intended the lockscreen as if unlecked, it automatically locks again after a while, and for this I used the same timer used by menu, but I wanted the button to act as a toggle to let the user disable lockscreen if needed.

Reply:
This "feature" isn't documented. And actually, your work isn't documented at all. That's bad; I wanted to postpone criticism for that to later - but it seems we need to have it now, to avoid further confusion about your work.
I think we should first have a simple "lock screen" without any timers. When we have that, we can talk about how to improve it, to have an option to auto-lock it after a while.
The way you hooked it into the menu timer is a bad idea.

Reply:
Ok, I'll go back to a not timed version of the feature and do one thing at time